src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -218,6 +280,13 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
             this.emit(MatrixRTCSessionEvent.MembershipsChanged, oldMemberships, this.memberships);
         }
 
+        const ts = (m: CallMembership): string => `${m.member.userId}:${m.deviceId}`;
```

Comment:
I'm reading `ts` as `timestamp` but this does not appear to be what this variable is...

It's also not dependent on anything from the current scope, so why not make it a separate function at the top level to assert its independence?